modular_citadel/code/game/gamemodes/gangs/gang_items.dm

```diff
@@ -0,0 +1,410 @@
+	var/item_path
+	var/cost
+	var/spawn_msg
+	var/category
+	var/list/gang_whitelist = list()
+	var/list/gang_blacklist = list()
+	var/id
+
+/datum/gang_item/proc/purchase(mob/living/carbon/user, datum/team/gang/gang, obj/item/device/gangtool/gangtool, check_canbuy = TRUE)
+	if(check_canbuy && !can_buy(user, gang, gangtool))
+		return FALSE
+	var/real_cost = get_cost(user, gang, gangtool)
+	if(!spawn_item(user, gang, gangtool))
+		gang.adjust_influence(-real_cost)
+		to_chat(user, "<span class='notice'>You bought \the [name].</span>")
+		return TRUE
+
+/datum/gang_item/proc/spawn_item(mob/living/carbon/user, datum/team/gang/gang, obj/item/device/gangtool/gangtool) // If this returns anything other than null, something fucked up and influence won't lower.
+	if(item_path)
+		var/obj/item/O = new item_path(user.loc)
+		user.put_in_hands(O)
+	else
+		return TRUE
+	if(spawn_msg)
+		to_chat(user, "[spawn_msg]")
+
+/datum/gang_item/proc/can_buy(mob/living/carbon/user, datum/team/gang/gang, obj/item/device/gangtool/gangtool)
+	return gang && (gang.influence >= get_cost(user, gang, gangtool)) && can_see(user, gang, gangtool)
+
+/datum/gang_item/proc/can_see(mob/living/carbon/user, datum/team/gang/gang, obj/item/device/gangtool/gangtool)
+	return TRUE
+
+/datum/gang_item/proc/get_cost(mob/living/carbon/user, datum/team/gang/gang, obj/item/device/gangtool/gangtool)
+	return cost
+
+/datum/gang_item/proc/get_cost_display(mob/living/carbon/user, datum/team/gang/gang, obj/item/device/gangtool/gangtool)
+	return "([get_cost(user, gang, gangtool)] Influence)"
+
+/datum/gang_item/proc/get_name_display(mob/living/carbon/user, datum/team/gang/gang, obj/item/device/gangtool/gangtool)
+	return name
+
+/datum/gang_item/proc/get_extra_info(mob/living/carbon/user, datum/team/gang/gang, obj/item/device/gangtool/gangtool)
+	return
+
+///////////////////
+//CLOTHING
+///////////////////
+
+/datum/gang_item/clothing
+	category = "Purchase Gang Clothes (Only the jumpsuit and suit give you added influence):"
+
+/datum/gang_item/clothing/under
+	name = "Gang Uniform"
+	id = "under"
+	cost = 1
+
+/datum/gang_item/clothing/under/spawn_item(mob/living/carbon/user, datum/team/gang/gang, obj/item/device/gangtool/gangtool)
+	if(gang.inner_outfits.len)
+		var/outfit = pick(gang.inner_outfits)
+		if(outfit)
+			var/obj/item/O = new outfit(user.loc)
+			user.put_in_hands(O)
+			to_chat(user, "<span class='notice'> This is your gang's official uniform, wearing it will increase your influence")
+			return
+	return TRUE
+
+/datum/gang_item/clothing/suit
+	name = "Gang Armored Outerwear"
+	id = "suit"
+	cost = 1
+
+/datum/gang_item/clothing/suit/spawn_item(mob/living/carbon/user, datum/team/gang/gang, obj/item/device/gangtool/gangtool)
+	if(gang.outer_outfits.len)
+		var/outfit = pick(gang.outer_outfits)
+		if(outfit)
+			var/obj/item/O = new outfit(user.loc)
+			O.armor = O.armor.setRating(melee = 20, bullet = 35, laser = 10, energy = 10, bomb = 30, bio = 0, rad = 0, fire = 30, acid = 30)
```

Comment:
armor teak 
25 melee 20 laser would help gangs survive sec raids better

Reply:
I don't think they should resist lasers a ton seeing as they are ment to fight each other, but I did add 5 to melee and only 5 to laser resists.